llvm/include/llvm/CodeGen/ValueTypes.td

```diff
@@ -338,6 +338,8 @@ def amdgpuBufferFatPointer : ValueType<160, 234>;
 // FIXME: Remove this and the getPointerType() override if MVT::i82 is added.
 def amdgpuBufferStridedPointer : ValueType<192, 235>;
 
+def vi8       : ValueType<8,  236>;  // 8-bit integer in FPR (AArch64)
```

Comment:
Working on the principle that we need a new MVT, let's go with `aarch64mfp8` to match the type we've added to clang.